be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -659,6 +732,9 @@ Status SegmentIterator::next_batch(RowBlockV2* block) {
         for (auto column_predicate : _col_predicates) {
             auto column_id = column_predicate->column_id();
             auto column_block = block->column_block(column_id);
+            if (column_predicate->type() == PredicateType::MATCH) {
```

Comment:
is there any chance to be true here?

Reply:
Because non vectorization does not support match query, in order to check match query must with vectorization